lib/bugsnag/breadcrumbs/breadcrumb.rb

```diff
@@ -0,0 +1,76 @@
+module Bugsnag::Breadcrumbs
+  class Breadcrumb
+    # @return [String] the breadcrumb name
+    attr_accessor :name
+
+    # @return [String] the breadcrumb type
+    attr_accessor :type
+
+    # @return [Hash, nil] metadata hash containing strings, numbers, or booleans, or nil
+    attr_accessor :meta_data
+
+    # @return [Symbol] set to `:auto` if the breadcrumb was automatically generated
+    attr_reader :auto
+
+    # @return [Time] a Time object referring to breadcrumb creation time
+    attr_reader :timestamp
+
+    ##
+    # Creates a breadcrumb
+    #
+    # This will not have been validated, which must occur before this is attached to a report
+    #
+    # @api private
+    #
+    # @param name [String] the breadcrumb name
+    # @param type [String] the breadcrumb type from Bugsnag::Breadcrumbs::VALID_BREADCRUMB_TYPES
+    # @param meta_data [Hash, nil] a hash containing strings, numbers, or booleans, or nil
+    # @param auto [Symbol] set to `:auto` if the breadcrumb is automatically generated
+    def initialize(name, type, meta_data, auto)
+      @should_ignore = false
+      self.name = name
+      self.type = type
+      self.meta_data = meta_data
+
+      # Use the symbol comparison to improve readability of breadcrumb creation
+      @auto = auto == :auto
+
+      # Store it as a timestamp for now
+      @timestamp = Time.now
```

Comment:
Use UTC

Reply:
We convert to utc before outputting it in the hash

Reply:
Do it here so you dont have different timezones in the library. Its a best practice to convert to UTC immediately.